src/server.js

```diff
@@ -68,6 +68,15 @@ app.post('/login', function (req, res) {
     });
 });
 
+app.get('/clusterName', function (req, res) {
+    var kc = new k8s.KubeConfig();
+    kc.loadFromDefault();
+    var outputRes = '{"clusterName":"' + kc.getCurrentCluster().name + '"}'
```

Comment:
you can construct the object and then make that as JSON.stringify() and send it.